src/bin/zmchatvidextziminstall

```diff
@@ -0,0 +1,409 @@
+#!/bin/bash 
+#
+# ***** BEGIN LICENSE BLOCK *****
+# Zimbra Collaboration Suite Server
+# Copyright (C) 2024 Synacor, Inc.
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software Foundation,
+# version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this program.
+# If not, see <https://www.gnu.org/licenses/>.
+# ***** END LICENSE BLOCK *****
+#
```

Comment:
Do we also need to enable other LDAP attributes zimbraFeatureBasicOneToOneChatEnabled, zimbraFeatureAdvancedChatEnabled, zimbraFeatureAdvancedChatVideoEnabled to TRUE after installation?

Reply:
admin will do this manually after installation. @prashantajari please correct me if I am wrong.

Reply:
@shrutig0510 these are account level features to be enabled for users on the required accounts as needed.